app/src/components/blocks/feedbackForm/FeedbackForm.tsx

```diff
@@ -0,0 +1,135 @@
+                        label="Tell us how we could improve this service or explain your experience using it. You
+                can also give feedback about a specific page or section in the service."
+                        rows={7}
+                        error={errors.feedbackContent?.message}
+                        {...feedbackContentProps}
+                    />
+                </Fieldset>
+
+                <Fieldset>
+                    <Fieldset.Legend size="m">
+                        How satisfied were you with your overall experience of using this service?
+                    </Fieldset.Legend>
+                    <Radios id="select-how-satisfied" error={errors.howSatisfied?.message}>
+                        {Object.values(SATISFACTION_CHOICES).map((choice) => (
+                            <Radios.Radio key={choice} value={choice} {...howSatisfiedProps}>
+                                {choice}
+                            </Radios.Radio>
+                        ))}
+                    </Radios>
+                </Fieldset>
+
+                <Fieldset>
+                    <Fieldset.Legend size="m">Leave your details (optional)</Fieldset.Legend>
+
+                    <p>
+                        If you’re happy to speak to us about your feedback so we can improve this
+                        service, please leave your details below.
+                    </p>
+
+                    <Input
+                        label="Your name"
+                        data-testid={FORM_FIELDS.respondentName}
+                        autoComplete="name"
+                        spellCheck={false}
+                        {...respondentNameProps}
+                    />
+
+                    <Input
+                        label="Your email address"
+                        hint="We’ll only use this to speak to you about your feedback"
+                        data-testid={FORM_FIELDS.respondentEmail}
+                        autoComplete="email"
+                        spellCheck={false}
+                        {...respondentEmailProps}
+                    />
+                </Fieldset>
+                {stage !== SUBMISSION_STAGE.Submitting ? (
+                    <Button type="submit">Send feedback</Button>
+                ) : (
+                    <SpinnerButton
+                        id="feedback-submit-spinner"
+                        status="Submitting..."
+                        disabled={true}
+                    />
+                )}
+            </form>
+            {/* to be removed when we got the confirmation page in place. */}
+            {result && (
+                <p>{`[Placeholder] called sendEmail() with data: \n${JSON.stringify(result)}`}</p>
+            )}
+        </div>
+    );
+}
+
+const renameRefKey = (props: UseFormRegisterReturn, newRefKey: string) => {
+    const { ref, ...otherProps } = props;
+    return {
+        [newRefKey]: ref,
+        ...otherProps,
+    };
+};
```

Comment:
Good idea, but can we move the creation of this to where the register callbacks happen. Can create some confusion to how react-hook-forms is working to some who know less UI, if it's at the bottom of the page